src/MIDebugEngine.sln

```diff
@@ -67,114 +58,160 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "prebuild", "prebuild\prebui
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Lab.Debug|Any CPU = Lab.Debug|Any CPU
+		Debug.Desktop|Any CPU = Debug.Desktop|Any CPU
 		Debug|Any CPU = Debug|Any CPU
 		Debug-PortablePDB|Any CPU = Debug-PortablePDB|Any CPU
+		Lab.Debug|Any CPU = Lab.Debug|Any CPU
 		Lab.Release|Any CPU = Lab.Release|Any CPU
+		Release.Desktop|Any CPU = Release.Desktop|Any CPU
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{4F38F986-5C93-44CB-A387-AEF3737E6D22}.Lab.Debug|Any CPU.ActiveCfg = Lab.Debug|Any CPU
-		{4F38F986-5C93-44CB-A387-AEF3737E6D22}.Lab.Debug|Any CPU.Build.0 = Lab.Debug|Any CPU
+		{4F38F986-5C93-44CB-A387-AEF3737E6D22}.Debug.Desktop|Any CPU.ActiveCfg = Debug.Desktop|Any CPU
+		{4F38F986-5C93-44CB-A387-AEF3737E6D22}.Debug.Desktop|Any CPU.Build.0 = Debug.Desktop|Any CPU
 		{4F38F986-5C93-44CB-A387-AEF3737E6D22}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Please try this test:
1. Make a second scratch enlistment
2. Pull down your changes
3. Open the .sln

Question: What is the default configuration?

VS seems to sort these alphabetically to decide what the default should be. So you need to pick a name which is after 'Debug' and I think 'Debug.Desktop' is going to be treated as before. That is why it is Lab.Debug instead of 'Debug.Lab'. I think you will need to rename your configuration to 'Desktop.Debug'/'Desktop.Release' (or 'Mono' if that makes more sense)